requirements.txt

```diff
@@ -22,26 +22,26 @@ msal-extensions==0.3.1
 msal==1.16.0
 msrest==0.6.21
 msrestazure==0.6.4
-natsort==8.0.2
-oauthlib==3.1.1
+natsort==8.1.0
+oauthlib==3.2.0
 octodns==0.9.14
 packaging==21.3
 pluggy==1.0.0
 portalocker==2.3.2
 pprintpp==0.4.0
 py==1.11.0
 pycountry-convert==0.7.2
-pycountry==20.7.3
+pycountry==22.1.10
```

Comment:
Not sure if we need these really.

Reply:
It's an downstream requirement pulled in through https://github.com/octodns/octodns/blob/fe7f96e2158c7c49bbe317cdc161088330d1077c/setup.py#L85. 

Looking at it now. It appears to be a dev requirement of octoDNS core, [`./script/generate-geo-data`](https://github.com/octodns/octodns/blob/1ef140f372bc38b3523f457c966aaa92e890808b/script/generate-geo-data#L5), and some of the individual providers require it. I'll look at moving it in octoDNS core and adding it to those providers, but atm this is correct.

Reply:
Interestingly, looking more closely those providers require pycountry-convert and not the base pycountry, e.g. https://github.com/octodns/octodns-constellix/blob/1f4f4952ff164b2e3dbe3aa2ba89994f9fc0a588/octodns_constellix/__init__.py#L8

Reply:
As of https://github.com/octodns/octodns/pull/869 (and some follow ons for specific providers) this will no longer be a runtime requirement of octoDNS. Once a new version of octoDNS is released and `update-requirements` is run here that will clear up.